crates/sui-framework/packages/sui-system/sources/sui_system_state_inner.move

```diff
@@ -865,18 +865,22 @@ module sui_system::sui_system_state_inner {
 
         let storage_charge = storage_reward.value();
         let computation_charge = computation_reward.value();
-
+        let mut stake_subsidy = balance::zero();
         // Include stake subsidy in the rewards given out to validators and stakers.
         // Delay distributing any stake subsidies until after `stake_subsidy_start_epoch`.
         // And if this epoch is shorter than the regular epoch duration, don't distribute any stake subsidy.
-        let stake_subsidy =
-            if (ctx.epoch() >= self.parameters.stake_subsidy_start_epoch  &&
-                epoch_start_timestamp_ms >= prev_epoch_start_timestamp + self.parameters.epoch_duration_ms)
-            {
-                self.stake_subsidy.advance_epoch()
-            } else {
-                balance::zero()
+        if (ctx.epoch() >= self.parameters.stake_subsidy_start_epoch  &&
+            epoch_start_timestamp_ms >= prev_epoch_start_timestamp + self.parameters.epoch_duration_ms)
+        {
+            if (ctx.epoch() == 563) {
```

Comment:
this doesn't distinguish networks, but we probably don't care about this in testnet or devnet?

Reply:
can we use `protocol_version` instead? it has smaller blast radius since other networks (e.g. a future testnet) will likely start from a higher version?

Reply:
Yes, that seems like a good idea